salad.html

```diff
@@ -40,72 +25,130 @@ <h1>🥗HAESNI'S SALAD🥗</h1>
       <cardsection>
         <div class="card">
           <h3>아보카도쉬림프샐러드</h3>
-          <h5>#생아보카도 #버터새우 #케이준버터</h5>
-          <img class="stuff" src="./assets/saladavocado.jpg" />
-          <img class="btn" src="./assets/heartbtn.png" />
+          <div class="hashtag">
+            <h5>#생아보카도 #버터새우 #케이준버터</h5>
+            <button>+</button>
+          </div>
+          <img
+            class="stuff"
+            src="./assets/saladavocado.jpg"
+            alt="아보카도쉬림프샐러드"
+          />
+          <img class="minebtn" src="./assets/heartbtn.png" alt="찜" />
         </div>
 
         <div class="card">
           <h3>치킨텐더샐러드</h3>
-          <h5>#닭안심 #머스타드소스</h5>
-          <img class="stuff" src="./assets/saladchiken.jpg" />
-          <img class="btn" src="./assets/heartbtn.png" />
+          <div class="hashtag">
+            <h5>#닭안심 #머스타드소스</h5>
+            <button>+</button>
+          </div>
+          <img
+            class="stuff"
+            src="./assets/saladchiken.jpg"
+            alt="치킨텐더샐러드"
+          />
+          <img class="minebtn" src="./assets/heartbtn.png" alt="찜" />
         </div>
 
         <div class="card">
           <h3>와사비크래미샐러드</h3>
-          <h5>#와사비크림 #크래미</h5>
-          <img class="stuff" src="./assets/saladcrap.jpg" />
-          <img class="btn" src="./assets/heartbtn.png" />
+          <div class="hashtag">
+            <h5>#와사비크림 #크래미</h5>
+            <button>+</button>
+          </div>
+          <img
+            class="stuff"
+            src="./assets/saladcrap.jpg"
+            alt="와사비크래미샐러드"
+          />
+          <img class="minebtn" src="./assets/heartbtn.png" alt="찜" />
         </div>
 
         <div class="card">
           <h3>파스타샐러드</h3>
-          <h5>#칠리 #바질 #파스타</h5>
-          <img class="stuff" src="./assets/saladpasta.jpg" />
-          <img class="btn" src="./assets/heartbtn.png" />
+          <div class="hashtag">
+            <h5>#칠리 #바질 #파스타</h5>
+            <button>+</button>
+          </div>
+          <img class="stuff" src="./assets/saladpasta.jpg" alt="파스타샐러드" />
+          <img class="minebtn" src="./assets/heartbtn.png" alt="찜" />
         </div>
 
         <div class="card">
           <h3>에그마요샌드위치</h3>
-          <h5>#에그마요 #유기농</h5>
-          <img class="stuff" src="./assets/sandeggmayo.jpg" />
-          <img class="btn" src="./assets/heartbtn.png" />
+          <div class="hashtag">
+            <h5>#에그마요 #유기농</h5>
+            <button>+</button>
+          </div>
+          <img
+            class="stuff"
+            src="./assets/sandeggmayo.jpg"
+            alt="에그마요샌드위치"
+          />
+          <img class="minebtn" src="./assets/heartbtn.png" alt="찜" />
```

Comment:
이부분은 시맨틱태그로 
```suggestion
<article>
               <header><h3>에그마요샌드위치</h3></header>
 </article>    
```
이런식으로도 적용할 수 있을 것 같아!

Reply:
네넵! 바꿨어요! 시멘틱 태그 공부 더 열심히 해야겠네용,,,